modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FieldValuesAssert.java

```diff
@@ -166,14 +164,15 @@ private static Map<String, String> _filterOnKey(
 			return map;
 		}
 
-		Stream<Map.Entry<String, String>> stream = SearchStreamUtil.stream(
-			map.entrySet());
+		Map<String, String> filteredMap = new HashMap<>();
```

Comment:
@julschong Please try to rethink about Brian's naming habit, usually we don't name a variable xxxMap or xxxList or xxxSet if we can give it other names.